steps/slack-step-get-member-id-by-email/step.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: integration/v1
+kind: Step
+name: slack-step-get-member-id-by-email
```

Comment:
Are we still including the initial `slack-step-*` stuff also? I was under the impression that this was for the container name, but steps and triggers excluded this.